src/tooling/debugging/openocd.md

```diff
@@ -0,0 +1,22 @@
+
+# OpenOCD
+
+Similar to [probe-rs](./probe-rs.md), OpenOCD does not have support for the Xtensa architecture. However, Espressif does maintain a fork of OpenOCD under [espressif/openocd-esp32](https://github.com/espressif/openocd-esp32) which has support for Espressif's chips.
+
+Instructions on how to install `openocd-esp32` for your platform can be found in [the espressif documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/api-guides/jtag-debugging/index.html#setup-of-openocd).
+
+## Setup for Espressif chips
+
+<!-- how to choose interface & chip -->
+
+Once installed, its as simple as running `openocd` with the correct scripts. For chips with the builtin usb jtag, there is normally a config that will work out of the box, for example on the esp32c3:
```

Comment:
```suggestion
Once installed, it's as simple as running `openocd` with the correct scripts. For chips with the builtin USB JTAG, there is normally a config that will work out of the box, for example on the ESP32-C3:
```

Just some grammar/formatting changes here. The `esp32c3` form is usually used for identifiers where as the `ESP32-C3` form is the "display" format, as per Espressif's preferences.